package.json

```diff
@@ -43,8 +43,10 @@
     "@fortawesome/free-brands-svg-icons": "^5.14.0",
     "@fortawesome/free-solid-svg-icons": "^5.14.0",
     "@fortawesome/react-fontawesome": "^0.1.11",
+    "bluebird": "^3.5.1",
```

Comment:
The need for the bluebird library has been removed in a previous PR, and we shouldn't need the package anymore.